app/src/main/scala/org/alephium/explorer/api/model/ExportType.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.api.model
+
+sealed trait ExportType
+
+object ExportType {
+  case object CSV  extends ExportType
+  case object JSON extends ExportType
```

Comment:
JSON is a very bloated data type, we may should not support it for exporting. It's also not for data analysis.

Reply:
as discussed on slack, let's use CSV only